domain/src/test/java/org/oppia/android/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -3398,6 +3398,34 @@ class ExplorationProgressControllerTest {
       .isEqualTo("Fractions")
   }
 
+  @Test
+  fun testFlashback_submitWrongRatioAnswer_onContinueFlashbackConfirmationDialog_returnsEphemeralStateWithFlashbackStateTrue() { // ktlint-disable max-line-length
```

Comment:
```suggestion
  testFlashback_submitWrongAnswer_moveToFlashbackState_returnsEphemeralStateWithFlashbackStateTrue {
```

Still descriptive while not exceeling max line length.

Reply:
Thanks for suggestion. But still exceeding max length.